src/components/organisms/ConversationWindow.vue

```diff
@@ -1,33 +1,25 @@
 <template>
-    <div class="w-full h-full flex flex-col">
-        <div class="flex border-b border-gray-infomd p-2">
-            <img src="../../assets/UserAccount/VirtualConcierge.svg" alt="Bot Image." class="h-10 mt-auto w-10">
```

Comment:
Some of this didn't merge well. The bot logo is missing and the alignment is off